### DISCUSSION TITLE
Consistent Attachments and Links

### DISCUSSION THREAD

**Discussion by evolve2k** [evolve2k/obsidian-pandoc-academic-word-doc-guide#5] [General]
@growingkittens [on HN](https://news.ycombinator.com/reply?id=40481330&goto=item%3Fid%3D40459404%2340481330) writes: 

"Thank you very much for this guide.
I don't know if this falls under your project scope, but I have a suggestion.
By default, Obsidian saves links in notes using the "shortest path" , which reduces compatibility: the links don't work in anything but Obsidian.

I found two plugins that manage relative links:
https://github.com/val3344/obsidian-update-relative-links
[https://github.com/dy-sh/obsidian-consistent-attachments-and...](https://github.com/dy-sh/obsidian-consistent-attachments-and-links)

**Comment by evolve2k** [comment]
Seems that Obsidian is taking shortcuts on the markdown standard in how it processes links (slyly making their tool harder to leave). 

I like the look of that second library and it's framing around making 'Consistent Attachments and Links'.
Practically what this seems to offer is that if you save your library to Github or serve the markdown on the web you'll want to have more explicit link references.

Wondering out loud, I'm wondering if this will also address the image referencing issue that's currently on the README of this guide as needing a workaround. 

Would love any thoughts on how this may help address [the embed images issue](https://github.com/evolve2k/obsidian-pandoc-academic-word-doc-guide?tab=readme-ov-file#2-embeds-your-images)

**Comment by growingkittens** [comment]
I think these plugins are important for archival concerns, to preserve the academic process.

I vaguely remember seeing some image management plugins talk about image embed issues. I will check my spreadsheet of plugins to find more clues.

I would like to offer more than clues for help, but I am not enough of a technical person to be able to jump right into an issue.
